src/nodes/ImageView.vue

```diff
@@ -243,6 +243,11 @@ export default {
 	methods: {
 		async loadPreview() {
 			this.attachment = await this.$attachmentResolver.resolve(this.src)
+			if (!this.attachment.previewUrl) {
+				this.loaded = true
+				this.failed = true
+				return
```

Comment:
Maybe that could be the better approach, so `this.loadPreview().catch(this.onImageLoadFailure)` could handle it?

```suggestion
				throw new Error('Attachment source was not resolved')
```

Reply:
Also works and sounds better, yeah